src/Component/Metadata/Resource/Factory/AttributesResourceMetadataCollectionFactory.php

```diff
@@ -122,6 +122,12 @@ private function getOperationWithDefaults(ResourceMetadata $resource, Operation
     {
         $resourceConfiguration = $this->resourceRegistry->get($resource->getAlias());
 
+        if (null === $operation->getTemplate()) {
+            $templateDir = $resource->getTemplatesDir() ?? '';
+            $template = sprintf('%s/%s.html.twig', $templateDir, $operation->getShortName() ?? '');
```

Comment:
Do we really want to search for templates in `/create.html.twig` and sometimes even `/.html.twig`? I would suggest we go with this behaviour only if `getTemplatesDir` is set. And if we want to provide some default folder for templates, it should be part of the configuration of ResourceBundle

Reply:
default template dir could be the the resource name.
Example: Resource(name: 'book') will have book as default template dir

I don't want any Resource configuration on the bundle itself to configure the resource at the end.

Reply:
But then, if you would like to use the default configuration, you are forced to duplicate templates from default folders (as every default folder is different). We need something better. Perhaps a super simple template in ResourceBundle? I'm still struggling with how to solve the issue properly for cases like right now Sylius show template (which does not exists and magically asks you for it)